docker.go

```diff
@@ -29,6 +29,8 @@ type Docker struct {
 	}
 
 	Pull DockerPull
+	Copy DockerCopy
+	Exec DockerExec
```

Comment:
Can we namespace these under the `Container` subgroup? Both `cp` and `exec` operate on a container that is already running, and the `docker` command also has them nested under its own `container` subcommand (I believe they exist at the top-level for backwards compatibility).

Reply:
Good find about the subcommands. I'll move them.

Reply:
I've also put them in at the correct alphabetical order since that seems preferred